sort.php

```diff
@@ -39,8 +54,10 @@ function bubbleSort($sort_array)
 
 $optind = null;
 $options = getopt("n", [], $optind);
-if(! empty($optind)) {
+if(! empty($options) && $argc > 2) {
     $fileName = $argv[$optind];
+} elseif(! getopt("n") && $argc === 2) {
+    $fileName = $argv[1];
 } else {
     $fileName = 'php://stdin';
 }
```

Comment:
Reduce this if/elseif/else to an if/else by checking whether `$argv[$optind]` (file name) is set.

Reply:
Use isset, not empty.